sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2167,6 +2167,19 @@ object functions {
     new Murmur3Hash(cols.map(_.expr))
   }
 
+  /**
+   * Calculates the hash code of given columns using the 64-bit
+   * variant of the xxHash algorithm, and returns the result as a long
+   * column.
+   *
+   * @group misc_funcs
+   * @since 2.4.1
+   */
+  @scala.annotation.varargs
+  def xxhash64(cols: Column*): Column = withExpr {
```

Comment:
We don't need `seed` in arguments?

Reply:
The `hash` function doesn't currently have a `seed` argument either.

In any case, I asked about this on dev@spark.apache.org ("[SQL] hash: 64-bits and seeding"), but didn't get any response to that part of my proposal (just the xxhash bit). I think if there was one it would have to come first, because the var args have to come last, something like the following?

```scala
def hash(seed: Int, cols: Column*): Column
// or, maybe, don't perpetuate the "bad"/non-specific name:
def murmur3(seed: Int, cols: Columns*): Column
```

```scala
def xxhash64(seed: Long, cols: Column*): Column
```

Reply:
Ah, I see. Its ok as it it.